.github/workflows/mobile-build.yml

```diff
@@ -0,0 +1,93 @@
+name: Build/mobile
+
+on: workflow_dispatch
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  CI: ''
+  SKIP_SIGNING: true
```

Comment:
```suggestion
```

We don't need the SKIP_SIGNING as that only relates to electron packaging. I'm not sure what clearing the CI var does, but I suspect it might not be needed either?

Reply:
Commented out for now.